torch/csrc/jit/passes/utils/memory_dag.h

```diff
@@ -38,6 +41,10 @@ class MemoryDAG {
   bool mayAlias(const Element* a, const Element* b) const;
   bool mayAlias(Element* a, Element* b) const;
 
+  // Does container hold reference to any memory that is stored in elem ?
+  bool mayContainAlias(const Element* elem, const Element* container) const;
```

Comment:
The implementation of this is commutative but the argument names and comment make it seem like it's not. Can we change it to `mayContainAlias(a, b)` and update the comment to make it clear that the order of the arguments does not matter?